server/src/model/Items.ts

```diff
@@ -10,7 +10,7 @@ import {
   Unique,
 } from "typeorm";
 // eslint-disable-next-line import/no-cycle
-import { Supplier, Comment, Rating, OrderItem, ItemTag } from ".";
+import { Supplier, Comment, Rating, OrderItem, ItemTag, ItemPicture } from ".";
 
```

Comment:
Was thinking that we shouldn't allow to have an item without a customer, would like to hear your opinion on that